docs/data/material/design-resources/material-ui-for-figma/material-ui-for-figma.md

```diff
@@ -100,3 +100,15 @@ If you need to replace a single component that's been updated, there are a coupl
 ## Feedback and bug reports
 
 If you've got any feedback, we'd love to [hear from you](https://github.com/mui/mui-design-kits/discussions/84).
+
+## Integrations
+
+### Quest
+
+[Quest](https://www.quest.ai/) provides a native integration with this design kit.
+
+When you design your components with the kit, you can use [Quest plugin](https://www.figma.com/community/plugin/862039267149408972/figma-to-react-from-quest) to convert your Figma designs into Material UI code. 
+
```

Comment:
```suggestion
```

same as above, it's feel like it's the same content, what is Quest about.

Reply:
Oh, right, let me fix it! Thanks @samuelsycamore

Reply:
I have restored the first blank line. It's only this one that felt too much.